cmd/evm/runner.go

```diff
@@ -280,7 +280,7 @@ func runCmd(ctx *cli.Context) error {
 			os.Exit(1)
 		}
 		if err := pprof.WriteHeapProfile(f); err != nil {
-			fmt.Println("could not write memory profile: ", err)
+			fmt.Println("could not write memory profile")
```

Comment:
why remove err?

Reply:
This is a strange one. `snyk` detects it as sensible info. It's actually a false leak. I'll restore it.

Reply:
Solved in [b55421c95](https://github.com/maticnetwork/bor/pull/578/commits/b55421c95d89726588a9c221f3d87358cfeb3a90)